src/app/services/security/http-interceptor.service.ts

```diff
@@ -22,7 +22,8 @@ export class HttpinterceptorService implements HttpInterceptor {
 
     if (tempToken != null) {
       const afterTokenreq: HttpRequest<any> = req.clone({
-        setHeaders: { authorization: tempToken },
+        withCredentials: true,
+        setHeaders: { 'x-csrf-token': tempToken }
       });
```

Comment:
This will work only for CSRF. A condition is needed for JWT as well.

Reply:
Ok